### DISCUSSION TITLE
Pathfinder 2e

### DISCUSSION THREAD

**Discussion by Deathe25** [etriebe/dnd-randomizer#106] [Ideas]
I was wondering if you could get it to work with Pathfinder 2e cuz I believe that that community would love it for the encounters it could make

**Comment by etriebe** [comment]
It does work with pf2e. Are you seeing errors that prevent it from working? @Deathe25

**Reply by Deathe25** [reply]
no i haven't tried it i just happen to see that it was a 5e module an was like i would love that for pathfinder
would it work with 3rd party content or just what's in the beastiarys

**Comment by etriebe** [comment]
Yep! It started as 5e but pf2e was added a number of years ago. Any local
content in compendiums should work!

**Reply by Deathe25** [reply]
i noticed that it wasnt usin monster from my Roll for combat compendiums just the stuff that paizo made

**Comment by etriebe** [comment]
Can you show what it says on compendium filter selection? Or save off
console logs after you create an encounter?
